ParkPlanner/Park.cs

```diff
@@ -183,7 +183,7 @@ public partial class PhoneNumber
     public partial class Entrance
     {
         [JsonProperty("cost")]
-        public string Cost { get; set; }
```

Comment:
3. Use decimal for Monetary Values:
     Use the decimal type instead of string for the Cost property in the Entrance class, if EntranceFees and EntrancePasses represent monetary values. 
     This provides better precision for monetary calculations.